plugins/polio/js/src/domains/VaccineModule/PerformanceDashboard/hooks/api/useGetPerformance.ts

```diff
@@ -0,0 +1,31 @@
+import { useSnackQuery } from 'Iaso/libs/apiHooks';
+import { useApiParams } from '../../../../../../../../../hat/assets/js/apps/Iaso/hooks/useApiParams';
+import { useUrlParams } from '../../../../../../../../../hat/assets/js/apps/Iaso/hooks/useUrlParams';
+import { getRequest } from '../../../../../../../../../hat/assets/js/apps/Iaso/libs/Api';
+
+import { PerformanceList } from '../../types';
+
+const getPerformanceDashboard = (params: any) => {
+    const queryString = new URLSearchParams(params).toString();
+    return getRequest(`/api/polio/performance_dashboard/?${queryString}`);
+};
+
+export const useGetPerformanceDashboard = (params: any) => {
+    const safeParams = useUrlParams(params);
+    const apiParams = useApiParams(safeParams);
+
+    return useSnackQuery<PerformanceList>(
+        [
+            'performance-dashboard',
+            apiParams,
+            apiParams.page,
+            apiParams.limit,
+            apiParams.order,
+        ],
```

Comment:
```suggestion
        [ 'performance-dashboard', apiParams ],
```

react-query is smart enough to detect changes in the object's entries